packages/fetch-and-compile/test/fixture.js

```diff
@@ -200,6 +200,80 @@ const etherscanFixture = {
     }
   },
 
+  "https://api-optimistic.etherscan.com/api": {
+    "0xca5f37e6D8bB24c5A7958d5eccE7Bd9Aacc944f2": {
+      status: "1",
+      message: "OK",
+      result: [
+        {
+          SourceCode: fs.readFileSync(
```

Comment:
I suppose this isn't really related to this PR specifically, but I thought we were trying to use `fse` instead of `fs` now for consistency?

Reply:
Hm, really?  My understanding was the opposite if anything -- we were trying to get rid of the `fs-extra` dependency, and just use `fs`, where possible.

...or are you just talking about the variable naming?  I don't see any reason to call it `fse` if it's just `fs` and not `fs-extra`...